apps/sms/views/conversation/js/compose.js

```diff
@@ -143,22 +138,19 @@ var Compose = (function() {
     state.emptyMessage = isEmptyMessage;
 
     Compose.updateEmptyState();
-    Compose.updateSendButton();
     Compose.updateType();
+    Compose.updateSendButton();
+    Compose.updateAttachButton();
```

Comment:
Okay, thanks for warning me, I'll see if it works without changing of order just to be on the safe side, otherwise - will check carefully.